CouplingDataUser.H

```diff
@@ -13,7 +13,8 @@ enum class LocationType
 {
     none,
     faceCenters,
-    faceNodes
+    faceNodes,
+    volume
```

Comment:
Would it make more sense to call this `cellCenters`, as I think you have it elsewhere? I am not sure.

Reply:
Maybe something like `volumeCenters`? I think it would make sense to keep "volume" in the name, maybe combine the two ideas :smile:

Reply:
I like the name! Go ahead.